source/modules.ts

```diff
@@ -0,0 +1 @@
+declare module 'meteor/caching-compiler';
```

Comment:
It's better to name the files just like the modules. Here it'd be `meteor_caching-compiler`.